lib/assets/javascripts/dmproadmap/forms.js

```diff
@@ -49,7 +49,7 @@ function validatePassword(sPassword) {
 
 // ---------------------------------------------------------------------------
 function validateEmail(sEmail) {
-  var filter = /^[a-zA-Z0-9]+[a-zA-Z0-9_.-]+[a-zA-Z0-9_-]+@[a-zA-Z0-9]+[a-zA-Z0-9.-]+.[a-z]{2,4}$/;
+  var filter = /[^@\s]+@(?:[-a-z0-9]+\.)+[a-z]{2,}$/;
```

Comment:
Just checked. Looks like `app/validators/email_validator.rb` is using the same regex that contact_us uses:
`/\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i`